package.json

```diff
@@ -64,6 +64,7 @@
     "npm-run-all": "^4.1.5",
     "optimize-css-assets-webpack-plugin": "^5.0.1",
     "prettier": "^2.0.4",
+    "redux-devtools-extension": "^2.13.8",
     "redux-logger": "^3.0.6",
```

Comment:
will redux-logger still be needed?

Reply:
If we want, we can use redux-logger as fallback in case the browser doesn't have the extension?
I'd prefer not though

Reply:
me neither,

should we remove the dependency?

Reply:
yes remove the dependency from package.json